apps/checkout/src/sections/CheckoutForm/CheckoutForm.tsx

```diff
@@ -34,7 +34,7 @@ export const CheckoutForm = () => {
   } = useCheckoutFinalize();
 
   const isLoading = loading || authenticating;
-  const usePaymentProvidersProps = usePaymentMethods();
+  const usePaymentProvidersProps = usePaymentMethods(checkout?.channel?.id);
```

Comment:
I thought this project uses `strictNullChecks: true` 🤔

Reply:
It does. I've checked it and the issue is in `useCheckout` hook - it lies to us because it uses non-null assertion (`!`) in return type 🙃

I've added a ticket to our backlog because that change would require refactoring a lot of types in the app.